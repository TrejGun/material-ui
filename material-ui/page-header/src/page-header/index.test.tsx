import { IntlProvider } from "react-intl";
import { cleanup, render } from "@testing-library/react";
import { createTheme, ThemeProvider } from "@mui/material";

import { PageHeader } from "./index";

afterEach(cleanup);

const i18n = {
  "pages.test.title": "Title",
};

describe("<PageHeader />", () => {
  it("renders component", () => {
    const { asFragment } = render(
      <ThemeProvider theme={createTheme()}>
        <IntlProvider locale="en" messages={i18n}>
          <PageHeader message="pages.test.title" />
        </IntlProvider>
      </ThemeProvider>,
    );

    expect(asFragment()).toMatchSnapshot();
  });
});
