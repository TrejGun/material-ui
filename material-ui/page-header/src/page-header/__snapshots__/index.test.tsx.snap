// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PageHeader /> renders component 1`] = `
<DocumentFragment>
  <div
    class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-undefined PageHeader-header-1 css-9cyib4-MuiGrid-root"
  >
    <div
      class="MuiGrid-root MuiGrid-item PageHeader-wrapper-2 css-13i4rnv-MuiGrid-root"
    >
      <h2
        class="MuiTypography-root MuiTypography-body1 PageHeader-title-3 css-ahj2mt-MuiTypography-root"
      >
        Title
      </h2>
    </div>
    <div
      class="MuiGrid-root MuiGrid-item PageHeader-buttons-4 css-13i4rnv-MuiGrid-root"
    >
      <div
        class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-undefined css-16g5rpk-MuiGrid-root"
      />
    </div>
  </div>
</DocumentFragment>
`;
